students/pedro-acuña/prueba1.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <a name="Grogu" id="Grogu"></a>
+    <h1>Grogu siendo adorable</h1>
+    <img src="https://sm.ign.com/t/ign_latam/screenshot/default/babygrogu_tch1.2560.jpg" alt = "Grogu" width="600"/>
+    <h2>Por qué deberías ver The Mandalorian</h2>
+    <p><b>The Mandalorian</b> es una serie sobre el universo de <i>Star Wars</i> de estilo <u>Western.</u>
+    La serie trata de un cazarrecompensas Mandaloriano al cual seguimos por toda la galaxia sus aventuras por
+    diversos mundos donde la acción y la aventura no faltarán. También conoceremos a Grogu, un pequeño muy adorable
+    que nos enternecerá el corazón. </p>
+    <ul>
+        <li><a href = "https://www.google.es/">Google</a> </li>
+        <li><a href = "mailto: pedritociudad@hotmail.es">Email de contacto</a></li>
+        <li><a href= "tel:601244768">Teléfono de contacto</a></li>
+    </ul>
+    <table style="text-align:center;" border ="2">
+        
+        <tr>
+            <td>Tamaño del disco según fabricante</td>
+            <td>Tamaño real del disco en gigabytes</td>
+        </tr>
+        <tr>
+            <td>160 GB</td>
+            <td>149 GB</td>
+        </tr>
+        <tr>
+            <td>250 GB</td>
+            <td>232 GB</td>
+        </tr>
+        <tr>
+            <td>320 GB</td>
+            <td>500 GB</td>
+        </tr>
+        <tr>
+            <td>1000 GB (1TB)</td>  
+            <td>931 GB</td>
+        </tr>
+        <tr>
+            <td>1500 GB (1,5 1TB)</td>
+            <td>1396 GB</td>
+        </tr>
+        <tr>
+            <td>2000GB (2TB)</td>
+            <td>1862 GB</td>
+        </tr>
+        <tr>
+            <td>3000 GB (3TB)</td>
+            <td>2793 GB</td>
+        </tr>
+    </table>
+    <form " method="post">
+    <p>Nombre:<input type ="text" required></p>
+    <p>Contraseña:<input type="password" required></p>
+    <p>Fecha de nacimiento: <input type="date" required></p>
+    <p>Email de contacto: <input type="email" required ></p>
+    <label for="genero femenino">Mujer:</label><input type="checkbox">
```

Comment:
label `for` attributes should not have space and must match with an input elemement `id`